design/work_item_link_type.go

```diff
@@ -76,26 +76,9 @@ var workItemLinkTypeRelationships = a.Type("WorkItemLinkTypeRelationships", func
 	a.Description(`JSONAPI store for the data of a work item link type.
 See also http://jsonapi.org/format/#document-resource-object-relationships`)
 	a.Attribute("link_category", relationWorkItemLinkCategory, "The work item link category of this work item link type.")
```

Comment:
Hm, good question. I guess, the only place we currently need the actual combinations is when asking a work item about the source and target link types in which it can be used. Most of the time, the UI should not need the actual type combinations.